gateway/fixtures/deployment/tgi-nginx/entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# entrypoint.sh
+#
+# Usage example:
+#   docker run \
+#     -p 8080:80 \
+#     -e BEARER_TOKEN=SUPER_SECRET_TOKEN \
+#     my-tgi-image \
+    # --model-id microsoft/Phi-3.5-mini-instruct \
+    # --port 8081 \
+    # --max-input-length 1024 \
+    # --max-total-tokens 2048 \
+    # --max-batch-prefill-tokens 1024 \
+    # --quantize fp8
```

Comment:
This is messy. Move to a README.md. Also both `port` and `p`?

Reply:
will move, but p is an argument for Docker and port for TGI

Reply:
ah ok

Reply:
also realized we shouldn't leave the --port free for TGI since NGINX expects 8081 so fixed that